### DISCUSSION TITLE
Control several hosts

### DISCUSSION THREAD

**Discussion by Eirikr70** [containrrr/watchtower#1758] [Feature Requests] [answered]
Hello, I have seen that Watchtower can control another host. But can it control several hosts at the same time : its own host and one or several remoste hosts ? How would I achieve that ? Should I attach both the docker socket as a volume and another docker host as an environment variable ?

**Comment by BLACK4585** [comment]
I'm also interested in this!

**Comment by piksel** [comment, marked as answer]
It's not something that is currently supported. You would need to run one watchtower instance per host (but they could run on the same host themselves).

**Reply by Eirikr70** [reply]
Right ! This perfectly fills my need, that is to update containers on two hosts in the same period of time. Thanks a lot !

**Reply by Eirikr70** [reply]
Well, although the response fills my needs, I had no success in setting it up :
```
watchtower-vault  | time="2023-09-16T19:08:15+02:00" level=error msg="error during connect: Get \"http://192.168.1.xxx:2375\\\"/v1.25/containers/json?filters=%7B%22status%22%3A%7B%22restarting%22%3Atrue%2C%22running%22%3Atrue%7D%7D&limit=0\": dial tcp: lookup 192.168.1.xxx:2375\": no such host"
watchtower-vault  | time="2023-09-16T19:08:15+02:00" level=info msg="Waiting for the notification goroutine to finish" notify=no
```

**Reply by piksel** [reply]
It looks like you have extra quotes around the host, if you are using docker compose yaml and the list format for the environment variables, the quotes goes on the whole line:
```yaml
env:
  - "DOCKER_HOST: http://192.168.1.xxx:2375"
```
instead of 
```yaml
env:
  - DOCKER_HOST: "http://192.168.1.xxx:2375"
```
